ppr-ui/src/composables/mhrRegistration/useMhrValidations.ts

```diff
@@ -13,6 +16,8 @@ export const useMhrValidations = (validationState: any) => {
 
   /** Is true when all flags are true in specified section. */
   const getStepValidation = (section: MhrSectVal): boolean => {
+    // If section is not part of validation model return true
+    if (!validationState[section]) return true
     return Object.values(validationState[section].value).every(val => val)
```

Comment:
Makes it so if a component tries to get validation for a section that is not part of the current flow it automatically returns true.

For example: In the manufacture flow we auto populate the home owners section, so we don't check it in the validation. So if the HomeOwnersReview component tries to check its validation it will return true,